scripts/docker_entrypoint.sh

```diff
@@ -0,0 +1,125 @@
+#!/usr/bin/env bash
+set -eo pipefail # stop script execution if any errors are encountered
+
+# script help info
+function usage() {
+    cat <<USAGE
+
+    Usage: 
+        $0 build -b binary_path -c config_path [KB path]
+        $0 run -b binary_path -c config_path [sc-machine args]
+
+    Options:
+        build <PATH>:       Rebuilds KB from sources (provide absolute path to the source folder or repo.path file).
+        run <args>:         Starts sc-machine. Arguments passed to this command will be redirected to sc-machine binary. You can set sc-server options from common config.
+
+        Setting REBUILD_KB environment variable inside the container will trigger a KB rebuild. Setting custom starting point for sc-builder can be done using KB_PATH environment variable, "/kb" is used as a default KB_PATH.
+        CONFIG_PATH and BINARY_PATH environment variables can provide the respective settings if the use of flags is undesirable.
+        EXTENSIONS_PATH can be set to specify the path for extensions used by sc-machine.
+
+USAGE
+    exit 1
+}
+
+function rebuild_kb() {
```

Comment:
```suggestion
function build_kb() {
```
sometimes it is used as a rebuild tool, but overall it just builds knowledge base